indra/newview/app_settings/settings.xml

```diff
@@ -16071,16 +16071,16 @@
     <key>Value</key>
     <integer>1</integer>
   </map>
-    <key>RenderVintageMode</key>
+    <key>RenderDisableVintageMode</key>
```

Comment:
Name it`RenderEnableAdvancedPipelineFeatures` instead of inverse logic of disabling something to enable new features.

Reply:
We did this to make the feature table happy.  We may rename it later, but for now it's fine.